components/Chat/Chat.vue

```diff
@@ -1052,7 +1052,7 @@
         notify            : Chat.$states.notify,
         getIgnoreList     : Chat.$states.ignoreList,
         getMessage        : Chat.$states.message,
-        getRecieveMentionsInLocal : Chat.$states.recieveMentionsInLocal,
+        getReceiveMentionsInLocal : Chat.$states.receiveMentionsInLocal,
```

Comment:
I'm dyslexic AF.

Reply:
To be fair, this was added by me. Noticed the spelling seemed wierd while scrolling through some time ago